app/Console/Commands/ImportSignupsCommand.php

```diff
@@ -70,13 +73,17 @@ public function handle()
                     'northstar_id' => $missing_signup['northstar_id'],
                     'campaign_id' => $missing_signup['campaign_node_id'],
                     'campaign_run_id' => $missing_signup['campaign_run_id'],
-                    'source' => 'sms',
-                    'created_at' => $missing_signup['signup_created_at_timestamp'],
+                    'source' => 'phoenix-next',
+                    'created_at' => $missing_signup['signup_created_at_timestamp'] ? $missing_signup['signup_created_at_timestamp'] : Carbon::now(),
                 ]);
 
                 if ($signup->id % $logfreq == 0) {
                     info('rogue:signupimport: Created signup ' . $signup->id);
                 }
+
+                // Business Logic
```

Comment:
shouldn't se use the `SignupService` so that we can just handle this automatically?

Reply:
That makes good sense to me.

Reply:
@sbsmith86 @DFurnes we'd have to use `v2` because this needs to support backdating, so I wasn't sure if that was a path we wanted to go down

Reply:
Oh and some history! I believe that this originally did not use the signup service because we were backfilling millions of mobile commons signups and we did not want those to be sent to Customer.io at the time